CONTRIBUTING.md

```diff
@@ -9,20 +9,31 @@ See [LICENSE.md](LICENSE.md) for the full license.
 
 ## Contributing translations
 
-**IMPORTANT:** We have professional translations of the Express documentation into:
-- German
-- French
-- Italian
-- Spanish
-- Brazilian Portuguese
-- Japanese
-- Simplified Chinese
-- Traditional Chinese
-- Russian
-- Korean
-
-Therefore, we can no longer accept community translations for these languages, except for corrections
-to the existing translations and updates to align with changes or additions to the English documentation.
+We highly encourage community translations! We no longer have professional translations, and we believe in the power of our community to provide accurate and helpful translations.
+
+We currently support translations for the following languages:
```

Comment:
This wording might be confusing.  Maybe instead say "The documentation is translated into these languages:"

Reply:
Updated at 06cf782446b73bce39603b729f8e405a39507195